azext_iot/operations/_mqtt.py

```diff
@@ -77,6 +79,20 @@ def execute(self, data, properties={}, publish_delay=2, msg_count=100):
         from tqdm import tqdm
         try:
             for _ in tqdm(range(msg_count), desc='Device simulation in progress'):
+                device_twin = _iot_device_twin_show(self.target, self.device_id)
+                if device_twin:
+                    desired_properties = device_twin.get("properties").get("desired")
```

Comment:
This double `get()` usage will throw an exception if the first `get()` returns `None`.

Reply:
You can do something like this to avoid the exception `device_twin.get("properties", {}).get("desired", {})`   (note also modified the last `get()` to return an empty `dict` if the key doesn't exist).

Reply:
Will address this in the next PR